filament/backend/src/opengl/OpenGLContext.cpp

```diff
@@ -284,6 +284,7 @@ void OpenGLContext::setDefaultState() noexcept {
 
     if (ext.EXT_clip_cull_distance) {
         glEnable(GL_CLIP_DISTANCE0);
+        glEnable(GL_CLIP_DISTANCE1);
     }
```

Comment:
I'm a little concerned having the clip planes always enabled even when not needed; could they have an impact on performance or even cause bugs (in the vast majority of cases where we don't use stereo).
Maybe we should formally add a enable/disable for clip planes in the backend API? I guess that will depend on how does this work on Metal or Vulkan (we can do that in another PR).

Reply:
Yeah sounds good. Metal doesn't need to do anything to enable them. Not sure about Vulkan.